scripts/modelbench/report.py

```diff
@@ -64,6 +64,7 @@
 
 new_performance_regression=pd.DataFrame()
 new_failures=pd.DataFrame()
+new_performance_improvement=pd.DataFrame()

```

Comment:
Please add a new_fixes model list also for those crash models in reference but fixed in target.